index.bs

```diff
@@ -810,8 +1102,61 @@ Each {{WebRequestEvent}} has:
         1. [=list/Append=] |handler config| to |valid handlers|.
 
     1. Return |valid handlers|.
+
   </div>
 
+## Monkey Patches ## {#api-web-request-monkey}
```

Comment:
Does this broadly apply to all of Controlled Frame? I imagined it did. If so, this should probably move to the top-level Controlled Frame monkey patching section. WDYT?

Reply:
I thought we were going to have dedicated monkey patch sections for WebRequest, ContextMenu, and CF element. We can merge them all together if you want, but I like the idea of having WebRequest be standalone.

Reply:
Yes, I too would like to see separate monkey patch sections. Is this specific monkey patching item for frameId etc only used for WebRequest? I got the sense from the text that it may be used outside of WebRequest. If it's only used there, then I think leaving under WebRequest makes sense assuming that the setup/requirements for setting it are not dependent on Controlled Frame-specific semantics. Let me know if that doesn't make sense.

Reply:
Oh, I see what you're asking now. It's used in ContextMenu so I'll move it up.